src/provider/camunda-platform/utils/ValidationUtil.js

```diff
@@ -0,0 +1,51 @@
+const SPACE_REGEX = /\s/;
```

Comment:
This file can be removed.

Reply:
Hi @barmac , Thank you for your suggestions. I will update the commit message. And regarding the ValidationUtil, I agree on the common file. Can we have the file in a common location for both bpmn and Camunda usage? Wanted to check with you if moving the file to a common location will be apt since it will be referred by both. wanted to know your thoughts on that. Thanks!

Reply:
Sounds good to me! Thank you

Reply:
![image](https://github.com/bpmn-io/bpmn-js-properties-panel/assets/12576777/c55cea84-acf2-4803-9bc4-6d5fbb249c38)

Does the above structure look good? Please suggest.

Reply:
It could go to the "shared" directory, but this way is also ok.

Reply:
@barmac , I have updated the PR with the requested changes. Please take a look.